docs/annotations.md

```diff
@@ -0,0 +1,229 @@
+    <id, label, lineColor, lineWidth>  # Optional general shape properties
+    "center": [10.3, -40.0, 0],        # Coordinate.  Required
+    "width": 5.3,                      # Number >= 0.  Required
+    "height": 17.3,                    # Number >= 0.  Required
+    "rotation": 0,                     # Number.  Counterclockwise radians around normal.  Required
+    "normal": [0, 0, 1.0],             # Three numbers specifying normal.  Default is positive Z.  Optional
+    "fillColor": "rgba(0, 255, 0, 1)"  # String.  See note about colors.  Optional
+}
+```
+
+### Point
+
+```
+{
+    "type": "point",                   # Exact string.  Required
+    <id, label, lineColor, lineWidth>  # Optional general shape properties
+    "center": [123.3, 144.6, -123]     # Coordinate.  Required
+}
+```
+
+### Polyline
+
+```
+{
+    "type": "polyline",                # Exact string.  Required
+    <id, label, lineColor, lineWidth>  # Optional general shape properties
+    "points": [                        # At least two points must be specified
+      [5,6,0],                         # Coordinate.  At least two required
+      [-17,6,0],
+      [56,-45,6]
+    ],
+    "closed": true,                    # Boolean.  Default is false.  Optional
+    "fillColor": "rgba(0, 255, 0, 1)"  # String.  See note about colors.  Optional
+}
+```
+
+### Rectangle
+
+```
+{
+    "type": "rectangle",               # Exact string.  Required
+    <id, label, lineColor, lineWidth>  # Optional general shape properties
+    "center": [10.3, -40.0, 0],        # Coordinate.  Required
+    "width": 5.3,                      # Number >= 0.  Required
+    "height": 17.3,                    # Number >= 0.  Required
+    "rotation": 0,                     # Number.  Counterclockwise radians around normal.  Required
+    "normal": [0, 0, 1.0],             # Three numbers specifying normal.  Default is positive Z.  Optional
+    "fillColor": "rgba(0, 255, 0, 1)"  # String.  See note about colors.  Optional
+}
+```
+
+### Rectangle Grid
+
+A Rectangle Grid is a rectangle which contains regular subdivisions, such as that used to show a regular scale grid overlay on an image.
+
+```
+{
+    "type": "rectanglegrid",           # Exact string.  Required
+    <id, label, lineColor, lineWidth>  # Optional general shape properties
+    "center": [10.3, -40.0, 0],        # Coordinate.  Required
+    "width": 5.3,                      # Number >= 0.  Required
+    "height": 17.3,                    # Number >= 0.  Required
+    "rotation": 0,                     # Number.  Counterclockwise radians around normal.  Required
+    "normal": [0, 0, 1.0],             # Three numbers specifying normal.  Default is positive Z.  Optional
+    "widthSubdivisions": 3,            # Integer > 0.  Required
+    "heightSubdivisions": 4,           # Integer > 0.  Required
+    "fillColor": "rgba(0, 255, 0, 1)"  # String.  See note about colors.  Optional
```

Comment:
I doubt @law12019 is using a fill color at all in his use of rectanglegrids.  I could see wanting the whole region to have a color with a partial opacity.